issue #214 - link to ISO download page and coreos-installer docs (#216)
coreos/fedora-coreos-docs: modules/ROOT/pages/bare-metal.adoc

To install FCOS onto bare metal using the live ISO, follow these steps:

> - Take the latest ISO image from the https://getfedora.org/coreos/download?tab=metal_virtualized&stream=stable[download page] or using podman (download stable by default, see https://coreos.github.io/coreos-installer/cmd/download/[documentation] for options):

Reviewer: How about this?

```suggestion
- Download the latest ISO image from the https://getfedora.org/coreos/download?tab=metal_virtualized&stream=stable[download page] or with podman (see https://coreos.github.io/coreos-installer/cmd/download/[documentation] for options):
```

And then let's add `-s stable` to the `podman` options.  Hopefully that'll provide a hint on how to select alternate streams, without emphasizing it too much.

Author: better, thanks!

Reviewer: Hmm, it looks like you opted not to add `-s stable`?

Author: Ah, I considered the update to the CMD documentation as separate. I am not sure how I do that.

Reviewer: Oh, I just meant in the example immediately following this line.

Author: Ah, now I understand what you meant.

More relevant in this context would be the (also default) `-p metal`.
So I have added them both for consistency with the URL to the download page that I used.

> - Take the latest ISO image from the https://getfedora.org/coreos/download?tab=metal_virtualized&stream=stable[download page] or using podman (download stable by default, see https://github.com/coreos/coreos-installer/blob/master/docs/cmd/download.md[documentation] for options):

Reviewer 2: Can you use https://coreos.github.io/coreos-installer/cmd/download/ instead? Thanks!

Author: done, thanks for the feedback!
